tutorials/getting_started/semantic_parsing.md

```diff
@@ -0,0 +1,315 @@
+# The AllenNLP Semantic Parsing Framework
+
+_Semantic parsing_ is the task of mapping language to some kind of formal meaning representation.
+This meaning representation could be a logical statement in lambda calculus or lambda-DCS, a set of
```

Comment:
maybe add that Lamda DCS is a lisp-like language for interacting with a KB

Reply:
Might be okay to omit lambda-DCS altogether. "logical statement in lambda calculus" gets the point across.